tests/Nox.ClientApi.Tests/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Application.Commands/UpsertWorkplaceEnumerationsTranslationsCommand.g.cs

```diff
@@ -19,82 +19,74 @@
+		{
+			localizedEntity.Name = command.WorkplaceOwnershipLocalizedUpsertDto.Name;
+		}
+		else
 		{
-            var entity = localizedEntities.SingleOrDefault(e=>e.Id == Enumeration.FromDatabase(dto.Id) && e.CultureCode == CultureCode.From(dto.CultureCode));
-	        if(entity is not null)
-			{
-                entity.Name = dto.Name;
-                entities.Add(entity);
-            }
-			else
-			{
-				var e = new WorkplaceOwnershipLocalized {Id = Enumeration.FromDatabase(dto.Id), CultureCode = CultureCode.From(dto.CultureCode), Name = dto.Name};
-				await Repository.AddAsync(e, cancellationToken);
-				entities.Add(e);
-			}
-        }
+			localizedEntity = new WorkplaceOwnershipLocalized {Id = command.Id, CultureCode = command.CultureCode, Name = command.WorkplaceOwnershipLocalizedUpsertDto.Name};
+			await Repository.AddAsync(localizedEntity, cancellationToken);
+		}
 		
-		//Update Default in Entity 
-		command.WorkplaceOwnershipLocalizedDtos.Where(dto=>dto.CultureCode == DefaultCultureCode.Value).ForEach(dto =>
+		if(command.CultureCode == DefaultCultureCode)
 		{
-			var e = new WorkplaceOwnership { Id = Enumeration.FromDatabase(dto.Id), Name = dto.Name };			
+			var e = new WorkplaceOwnership { Id = command.Id, Name = command.WorkplaceOwnershipLocalizedUpsertDto.Name };			
 			Repository.Update(e);
-		});
+		}
+		
 		
 
-		await OnCompletedAsync(command, entities);
+		await OnCompletedAsync(command, localizedEntity);
 		await Repository.SaveChangesAsync(cancellationToken);
-		return command.WorkplaceOwnershipLocalizedDtos;
+		return new WorkplaceOwnershipLocalizedKeyDto(command.Id.Value, command.CultureCode.Value);
 	}
 }
-public class UpsertWorkplacesOwnershipsTranslationsCommandValidator : AbstractValidator<UpsertWorkplacesOwnershipsTranslationsCommand>
+public class UpsertWorkplacesOwnershipsTranslationCommandValidator : AbstractValidator<UpsertWorkplacesOwnershipsTranslationCommand>
 {
 	private static readonly int[] _supportedIds = new int[] { 1000, 4000, 5000, };
 	
-    public UpsertWorkplacesOwnershipsTranslationsCommandValidator(NoxSolution noxSolution)
+    public UpsertWorkplacesOwnershipsTranslationCommandValidator(NoxSolution noxSolution)
     {
-		RuleFor(x => x.WorkplaceOwnershipLocalizedDtos)
-			.Must(x => x != null && x.Count() > 0)
-			.WithMessage($"{nameof(UpsertWorkplacesOwnershipsTranslationsCommand)} : {nameof(UpsertWorkplacesOwnershipsTranslationsCommand.WorkplaceOwnershipLocalizedDtos)} is required.");
-		
-		RuleForEach(x => x.WorkplaceOwnershipLocalizedDtos)
-			.Must(x => noxSolution!.Application!.Localization!.SupportedCultures.Select(c => c.ToDisplayName()).Contains(x.CultureCode))
-			.WithMessage((_,x) => $"{nameof(UpsertWorkplacesOwnershipsTranslationsCommand)} : {nameof(UpsertWorkplacesOwnershipsTranslationsCommand.WorkplaceOwnershipLocalizedDtos)} contains unsupported culture code: {x.CultureCode}.");
+	    System.Diagnostics.Debug.WriteLine("UpsertTranslationCommandValidator");
```

Comment:
Was this added by mistake?

Reply:
It was for debug purposes during the implementation, removed.